### PR TITLE
created saga and reducer for addClass POST, server route for POST in …

### PR DESCRIPTION
**Pull request by Quickz03**
…progress

### CODE REVIEW

File: src/redux/reducers/addClassReducer.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const setClassReducer = (state = [], action) => {
```
Comment (jwhy89) [inline src/redux/reducers/addClassReducer.js]: Change the name of addClassReducer to setClassReducer

File: src/redux/reducers/index.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import instructorSchedule from './instructorScheduleReducer';
 import classCount from './classCountReducer';
 import session from './sessionReducer';
 import setInstructor from './setInstructorReducer';
+import setClassReducer from './addClassReducer';
```
Comment (jwhy89) [inline src/redux/reducers/index.js]: change to setClassReducer

**Review by jwhy89 — CHANGES_REQUESTED** [review]
Please make changes to the two comments below. Looks good so far.

File: src/redux/reducers/addClassReducer.js (hunk 1 above)
Comment (Quickz03) [reply src/redux/reducers/addClassReducer.js]: done

**Review by jwhy89 — APPROVED** [review]
Looks good 👍

**Pull request merged**